src/components/text-input/TextInput.root.tsx

```diff
@@ -57,12 +75,12 @@ const TextInputRoot = styled.div<PTextInputRoot>(
         return css`
             ${actionStyles};
             color: ${colors.text};
+            box-sizing: content-box;
```

Comment:
why is this needed?

Reply:
when we discussed the paddings with Michael there was a mismatch with the designs as in the size they wanted didn't include the borders. Also when we use tab-focus the position of the element switch slightly because we are adding the extra border. In order to keep the consistent padding with the values we specified and to prevent the slight shift, this fixes both issues.